openapi-plugin/src/main/java/io/micronaut/gradle/openapi/DefaultOpenApiExtension.java

```diff
@@ -104,7 +104,7 @@ private void configureCommonExtensionDefaults(OpenApiSpec spec) {
         spec.getUseOptional().convention(false);
         spec.getUseReactive().convention(true);
         spec.getSerializationFramework().convention(DEFAULT_SERIALIZATION_FRAMEWORK);
-        spec.getAlwaysUseGenerateHttpResponse().convention(false);
+        spec.getGenerateHttpResponseAlways().convention(false);
```

Comment:
:x: This is a breaking change, we can't do this. The grammar of `getGenerateHttpResponseAlways` is also quite strange.